source/_integrations/freedompro.markdown

```diff
@@ -11,6 +12,7 @@ ha_codeowners:
 ha_domain: freedompro
 ha_platforms:
   - light
+  - sensor
```

Comment:
As well as adding platform and categories, update the text below to explain what is supported:
> This integration lets you control all [Freedompro](https://freedompro.eu/) accessories.